### PR TITLE
Change years + add committee members

### PR DESCRIPTION
**Pull request by MaxxxvD**
I updated the committee page, made some year fixes and addressed this [issue](https://github.com/JuliaCon/www.juliacon.org/issues/466) and this [one](https://github.com/JuliaCon/www.juliacon.org/issues/464).

### CODE REVIEW

**Review by mkitti — APPROVED** [review]

**Comment by mkitti** [comment]
@pabloferz

**Comment by rayegun** [comment]
I would like to request no picture for myself at this time

**Comment by MaxxxvD** [comment]
No problem! I’ll remove it

**Review by pabloferz — COMMENTED** [review]
We actually need to ask everyone if they are OK with including their photo.

**Comment by MaxxxvD** [comment]
I went for the easier way by just reverting the changes and keep the list instead

File: 2024/committee.md [hunk 1]
```diff
@@ -1,17 +1,20 @@
-# JuliaCon 2023 Organizing Committee
+# JuliaCon 2024 Organizing Committee
 
 * Ranjan Anantharaman, co-Chair [Twitter](https://twitter.com/ranjan_ananth)
-* Raye Kimmerer, co-Chair & Local Chair [Twitter](https://twitter.com/OfficialLoganK)
+* Raye Kimmerer, co-Chair & Local Chair [Twitter](https://github.com/rayegun)
 * Katharine Hyatt, Program [Twitter](https://twitter.com/kslimes)
 * Michael Herbst, Program [Github](https://github.com/mfherbst)
 * Mark Kittisopikul, Finance [Twitter](https://twitter.com/markkitti)
```
Comment (mkitti) [inline 2024/committee.md:7]:
```suggestion
* Mark Kittisopikul, Finance [Twitter](https://twitter.com/markkitti), [Github](https://github.com/mkitti), [LinkedIn](www.linkedin.com/in/markkittisopikul)
```

**Review by bvdmitri — APPROVED** [review]

**Pull request merged**